core/src/test/java/org/infinispan/replication/ReplicatedAPITest.java

```diff
@@ -57,7 +57,9 @@ public void put() {
       cache1.put("key", "value");
       waitForRpc(cache2);
 
+      tm(cache1).begin();
```

Comment:
I was just abusing the test to debug explicit transactions with just one read operation - I should create a new test method instead.